README.md

```diff
@@ -17,30 +17,55 @@ And then execute:
 Or install it yourself as:
 
     $ gem install 'openstudio-metadata'
+	
+## Usage
+
+Find documentation at https://rubydoc.info/github/NREL/openstudio-metadata-gem/
 
 ## Note
 
 This Gem requires OpenStudio@3.0 or greater
```

Comment:
The hyperlink on this in rubydocs is a mailto

Reply:
I don't seem to see that, are you using `yard server`?

Reply:
On my local yard server this doesn't seem to happen, not sure what's causing it, but I'd probably need to troubleshoot it after merge if it's still a problem.